src/main/java/com/code_intelligence/jazzer/agent/RuntimeInstrumentor.kt

```diff
@@ -57,6 +57,17 @@ class RuntimeInstrumentor(
         protectionDomain: ProtectionDomain?,
         classfileBuffer: ByteArray,
     ): ByteArray? {
+        var pathPrefix = ""
+        if (Opt.instrumentOnly && protectionDomain != null) {
```

Comment:
I wonder if we should always do this. WDYT?

Reply:
You'll know more about the expected average use case here, but to me this makes sense to always keep these separated when running instrument_only. If I need my jars merged into one jar, I think I should do that before running --instrument_only command on my list of jars.

Reply:
Well, let's keep it as is for now then.